packages/backend-app-api/src/config/config.ts

```diff
@@ -89,30 +89,34 @@ export async function loadBackendConfig(options: {
     configRoot: paths.targetRoot,
     configTargets: configTargets,
     remote: options.remote,
-    watch: {
-      onChange(newConfigs) {
-        console.info(
-          `Reloaded config from ${newConfigs.map(c => c.context).join(', ')}`,
-        );
-        const configsToMerge = [...newConfigs];
-        if (options.additionalConfigs) {
-          configsToMerge.push(...options.additionalConfigs);
-        }
-        config.setConfig(ConfigReader.fromConfigs(configsToMerge));
-      },
-      stopSignal: new Promise(resolve => {
-        if (currentCancelFunc) {
-          currentCancelFunc();
-        }
-        currentCancelFunc = resolve;
+    watch: Boolean(args?.watch)
```

Comment:
Let's make this an option forwarded from `loadBackendConfig` instead. It's like to avoid adding a flag for this

Reply:
Done!